config/routes.rb

```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
   get 'welcome/home'
   root 'welcome#home'
```

Comment:
`WelcomeController#home` is available at 2 URL paths: `/` and `/welcome/home`. Aim for having one path per `controller#action` route. The `get 'welcome/home'` route can be safely deleted.